runtime/doc/usr_27.txt

```diff
@@ -83,7 +83,7 @@ matter if 'ignorecase' or 'smartcase' was changed.
 
 	Note:
 	If your search takes much longer than you expected, you can interrupt
-	it with CTRL-C on Unix and  CTRL-Break on MS-DOS and MS-Windows.
+	it with CTRL-C on Unix and CTRL-Break on Windows.
```

Comment:
`CTRL-C` doesn't work on Console Vim on Windows it appears. It appears to work in Gvim though.